package.json

```diff
@@ -1,5 +1,14 @@
 {
     "devDependencies": {
         "prettier": "2.4.1"
+    },
+    "dependencies": {
+        "axios": "^0.24.0",
+        "chalk": "^4.1.2",
+        "minimatch": "^3.0.4",
+        "nocamel": "^1.0.2",
+        "semver": "^7.3.5",
+        "ws": "^8.3.0",
+        "yargs": "^17.2.1"
```

Comment:
Whats with this?

Reply:
no idea lol it told me to add those but i removed them now